zerosum/zerosum.py

```diff
@@ -280,6 +282,21 @@ def zerosum(entries, options_map, config):
     return(new_open_entries + entries, errors)
 
 
+def mark_unmatched(entries, unused_options_map, config):
```

Comment:
This seems redundant to me, given all unmatched postings will remain in the source account. So this simply slows things down. Perhaps you could move this to its own plugin? Or at least allow it to be disabled (by default)?

Reply:
Say I have `Equity:Transfers` and `Equity:Transfers:Matched` accounts. This marks all the transactions in `Equity:Transfers` as WARNING. In Fava, with default options, you cannot view transactions that are *only* in `Equity:Transfers` - the view of `Equity:Transfers` includes all the transactions in `Equity:Transfers:Matched` as well. Marking transaction in the parent account as WARNING makes user identify the unmatched transactions easier.

But I agree with you, I'll make this toggle-able.

Reply:
That makes sense! Yes, please feel free to add a config option to the config dict (eg: 'flag_unmatched').

Also, would you mind renaming this function to 'flag_unmatched'?

Reply:
The function has been renamed. And it's default off :-)